examples/agent-bot/MyCoolAgent.js

```diff
@@ -73,6 +74,19 @@ class MyCoolAgent extends Agent {
                         });
                     });
                     this.subscribeMessagingEvents({dialogId: change.result.convId});
+                } else if(change.type === 'UPSERT' && openConvs[change.result.convId] && change.result.conversationDetails.participants.filter(p => p.role === 'CONSUMER')[0].id !== this.consumerId) {
+                    // Step Up...
```

Comment:
Will API consumers understand what 'step up' means?

Is it feasible that one day we may support a consumer (for example) logging in as one user, then logging out before logging in as another?

```suggestion
                    // ConsumerID changed. Typically, a Step Up from an unauthenticated to an authenticated user.
```

Reply:
Sounds like you are describing step down.
However, this is unlikely to be supported at a conversation level IMO.

The suggested comment change makes sense to me.